app/src/main/java/waiter/ClientConnection.java

```diff
@@ -0,0 +1,27 @@
+package waiter;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.io.PrintStream;
+import java.net.Socket;
+
+public class ClientConnection implements Connection {
+
+    private final Socket client;
+
+    ClientConnection(Socket client) {
+        this.client = client;
+    }
+    public Message read() throws IOException {
```

Comment:
```suggestion
    }
    
    public Message read() throws IOException {
```

Reply:
Resolved in [this commit](https://github.com/jonmanahan/waiter/pull/2/commits/e53fdbc1eec22480eae57e79838d0f1186c16f90), if it resolves the issues please hit resolve conversation if possible, thank you all a ton!